Services/Notifications/classes/class.ilObjNotificationAdmin.php

```diff
@@ -1,45 +1,62 @@
 <?php
```

Comment:
Please add `strict_types` (for every file in `Services/Notifications`).

Reply:
Same here. I would outsource this into a major refactoring/PHP8 PR